tests/console/redis.pm

```diff
@@ -0,0 +1,120 @@
+# SUSE's openQA tests
+#
+# Copyright (C) 2022 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
```

Comment:
Hm, this should have failed t/01_style.t in https://github.com/os-autoinst/os-autoinst-distri-opensuse/runs/4717337728?check_suite_focus=true#step:6:11

Please use the common SPDX license header style:

```suggestion
# Copyright SUSE LLC
# SPDX-License-Identifier: FSFAP
```

Reply:
Thanks for reviewing @okurz, I have made the suggested changes.